packages/core/botpress/src/server/secured.js

```diff
@@ -223,6 +223,8 @@ module.exports = (bp, app) => {
   })
 
   app.get('/api/community/hero', (req, res) => {
-    res.send({ hidden: bp.botfile.heroSection.hidden })
+    const hidden = _.get(bp, 'botfile.heroSection.hidden', false)
+
+    res.send({ hidden })
```

Comment:
@dmytropanontko , could it be one-liner? E.g. `res.send({ hidden: _.get(bp, 'botfile.heroSection.hidden', false) })`